### ISSUE TITLE
mysql >= 5.7.6 breaks (at least on CentOS)

### ISSUE THREAD

**Issue by rhertogh** [chef-cookbooks/mysql#309]
mysql-community-server 5.7.6 is missing mysqld_safe (at least in the CentOS package) causing a 'No such file or directory' error.

Reproduce (tested on CentOS 7)
```
mysql_service 'default' do
  port '3306'
  version '5.7'
  initial_root_password 'very secret password'
  action [:create, :start]
end

mysql_config 'default' do
  instance 'default'
  source 'default.conf.erb'
  version '5.7'
  action :create
	notifies :restart, 'mysql_service[default]'
end
```
Workaround

* Use version 5.6
* ~~Modify the mysql cookbook to stick to version 5.7.5 (this cannot be done by simply passing it in the LWRP)~~
* Use package_version (not tested)

**Comment by someara** [comment]
Do you know if this is a bug in the packaging? 
Or if they've removed the binary in purpose and now want people to initialize databases differently?

**Comment by someara** [comment]
Looks like they've pulled the binaries...
https://bugs.mysql.com/bug.php?id=75848

**Comment by someara** [comment]
PS: Does the `package_version` parameter on the mysql_service resource not work?

**Comment by rhertogh** [comment]
Missed that one, that probably works.

**Comment by someara** [comment]
Fixed in 3ce278d0b5be851b060c58468f52e3ee9d00a9d9
Released as 6.0.17
